troubleshooting/js-errors.md

```diff
@@ -135,3 +136,25 @@ This error indicates that [the app is using an old or wrong version of the `tele
 ## Object doesn't support property or method 'assign'
 
 Under IE, you may get errors similar to `Object doesn't support property or method 'assign'` or errors that relate other modern JS features that are not supported under IE. The reason is that we use modern code that may not work under IE - it is not one of the [browsers we support]({%slug system-requirements%}#browser-support), and WebAssembly does not work on it anyway, so modern Blazor apps won't run on IE regardless.
+
+## Maximum call stack size exceeed
+
+Some common causes for the `Maximum call stack size exceeed` error are:
+
+* Update from older to newer .NET version
+* Old version of the `Telerik.UI.for.Blazor` (4.6.0 and older) with .NET 8
+* All references to `Telerik.UI.for.Blazor` in your app are not for latest `Telerik.UI.for.Blazor` version 
+* The project is runnable in one browser and not in another
```

Comment:
Refactor and reword this section. The main statement should be that an old `telerik-blazor.js` file (<= 4.5.0) is used in a .NET 8 app. The reason for the situation can be upgrade, cache, etc.

In addition, on Mac I can reproduce the problem with 4.5.0 in .NET 6 too. This implies that .NET is using the latest SDK to build the app, no matter the .NET version in the app. This is a rare case, as most of our customers are on Windows, but we can also mention it.